features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Applications/Listing/Listing.jsx

```diff
@@ -198,81 +121,58 @@ class Listing extends Component {
     }
 
     /**
-     *
-     *
      * @memberof Listing
      */
     componentDidMount() {
         this.updateApps();
     }
 
     /**
-     *
-     *
      * @memberof Listing
      */
     updateApps = () => {
-        const promised_applications = Application.all();
-        promised_applications
+        const { history, location } = this.props;
+        const promisedApplications = Application.all();
+        promisedApplications
             .then((applications) => {
-                const apps = new Map(); // Applications list put into map, to make it efficient when deleting apps (referring back to an App)
+                // Applications list put into map, to make it efficient when deleting apps (referring back to an App)
+                const apps = new Map();
                 applications.list.map(app => apps.set(app.applicationId, app)); // Store application against its UUID
                 this.setState({ data: apps });
             })
             .catch((error) => {
                 if (process.env.NODE_ENV !== 'production') console.log(error);
-                const status = error.status;
+                const { status } = error;
                 if (status === 404) {
+                    // eslint-disable-next-line react/no-unused-state
                     this.setState({ notFound: true });
                 } else if (status === 401) {
+                    // eslint-disable-next-line react/no-unused-state
                     this.setState({ isAuthorize: false });
-                    const params = qs.stringify({ reference: this.props.location.pathname });
-                    this.props.history.push({ pathname: '/login', search: params });
+                    const params = qs.stringify({ reference: location.pathname });
+                    history.push({ pathname: '/login', search: params });
```

Comment:
Shall we replace the following;
```
                    this.setState({ isAuthorize: false });
                    const params = qs.stringify({ reference: location.pathname });
                    history.push({ pathname: '/login', search: params });
```
with 
`window.location = '/store-new/services/configs';`.

Reply:
This will redirect the user to the login page is s/he does not have an access token.  The 'isAuthorize' state is redundant since it's not used elsewhere.

Reply:
done